Assets/Scripts/Agent/NpcMove.cs

```diff
@@ -0,0 +1,233 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using Photon.Pun;
+
+
+public class NpcMove : MonoBehaviourPun
+{
+    [SerializeField]
+    // Transform _destination;
+    NavMeshAgent navMeshAgent;
+    public float wanderRadius = 30f;
+    public string current_animation_state;
+    public int frames = 0;
+    public Vector3 targetVector;
+    List<string> animationConditions = new List<string>() {
+      "Talking","Dancing","Idle"
+    };
+    public float speed = 0.5f;
+    Animator animator;
```

Comment:
Do we actually need an Animator here? In theory, all animations as a result of interaction should happen in the interactable (having called `PrimaryInteraciton`). You also seem to have a separate script for agent animations?